pkg/controllers/disruption/orchestration/suite_test.go

```diff
@@ -141,15 +140,12 @@ var _ = Describe("Queue", func() {
 		node1.Spec.Taints = append(node1.Spec.Taints, v1beta1.DisruptionNoScheduleTaint)
 		node2.Spec.Taints = append(node2.Spec.Taints, v1beta1.DisruptionNoScheduleTaint)
 
-		ncKey = nodeclaim.Key{
-			Name:      test.RandomName(),
-			IsMachine: true,
```

Comment:
Good thing we removed the conditional tainting here so that this didn't matter. :)